build/images.bzl

```diff
@@ -25,4 +25,9 @@ def define_base_images():
         repository = "distroless/static",
         digest = "sha256:a7752b29b18bb106938caefd8dcce8a94199022cbd06ea42268b968f35e837a8",
     )
-
+    container_pull(
+        name = "dynamic_base",
+        registry = "gcr.io",
+        repository = "distroless/base",
+        digest = "sha256:75f63d4edd703030d4312dc7528a349ca34d48bec7bd754652b2d47e5a0b7873",
+    )
```

Comment:
I don't undestand why this is called "dynamic base".
Is it because the "base" image contains glibc, libssl etc and allows entrypoint's to be dynamically linked?

As mentioned above, please add a note to the README about when a developer would want to compile with CGO and use the base rather than the static image.

Reply:
They would only want to compile with cgo if they were importing a C library. In that case they may need to create a dynamically linked binary and would need the 'dynamic' base. It _is_ a very specific use case though.